js/components/developer/developer-screen/index.js

```diff
@@ -16,6 +16,7 @@ export default class DeveloperScreen extends React.Component {
     return (
       <Container>
         <Content>
+          <Button onPress={() => navigate('JobInfoScreen')} title="Job Info Screen" />
```

Comment:
⚠️  Missing I18n. I know I'm nagging, but its such a pain of doing afterwards...